scripts/create_status_report.py

```diff
@@ -0,0 +1,231 @@
+import sys
+from datetime import datetime, timedelta
+import os
+import os.path
+
+STATUS_TARGET_NAME = 0
+STATUS_TARGET_LAST_COMLETED_DATE = 1
+STATUS_TARGET_STATUS = 2
+STATUS_TARGET_STATUS_DATE = 3
+STATUS_TARGET_DURATION = 4
+STATUS_TARGET_DEPENDANCIES = 5
+STATUS_TARGET_LOG_FOLDER = 6
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+BUFFER_OF_DAYS = 15
+
+dashboard_file_name = "index.html"
+tblStatus = []
+
+# ====================================================================================
+# home-brew relational DB interface
+# plain files pipe (|) separated
+# ====================================================================================
+def loadDBFile(strInputFile, encoding="utf-8", separator="|"):
+    cells = []
+    if os.path.exists(strInputFile):
+        filehandle = open(strInputFile, 'r', encoding=encoding)
+        txt = filehandle.readline().strip()
+        while len(txt) != 0:
+            if txt[0:1] != "#":
+                row = txt.strip().split(separator)
+                for i in range(len(row)):
+                    row[i] = row[i].strip()
+                if len(row) > 1:
+                    cells.append(row)
+            txt = filehandle.readline()
+        # end of while
+        filehandle.close()
+    return cells
```

Comment:
we don't need anything "homebrew" db related here

Reply:
Here we just read from text file. make_profiler.db is a plain space separated file.